torch/distributed/_tensor/redistribute.py

```diff
@@ -79,6 +79,7 @@ def _redistribute_with_local_tensor(
     current_placements: Sequence[Placement],
     target_placements: Sequence[Placement],
 ) -> torch.Tensor:
+    dtensor_log.debug("_redistribute_with_local_tensor")
```

Comment:
Don't need to be in this PR, but ideally we want to have the current and target placements logged so that users know what is happening here?